ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -996,6 +1025,10 @@ + (BOOL)supportsSecureCoding {
     return YES;
 }
 
+- (NSString*)stringForAnswer:(id)answer {
+    return ORKTimeOfDayStringFromComponents(answer);
+}
```

Comment:
Since this doesn't translate well with localization, we might want to refactor the logic for `stringFromAnswer:` for any of the date time answer formats.